packages/x-date-pickers/src/PickersShortcuts/PickersShortcuts.tsx

```diff
@@ -14,20 +15,24 @@ export interface PickersShortcutsItem<TValue> {
   getValue: (params: PickersShortcutsItemGetValueParams<TValue>) => TValue;
 }
 
-export interface PickersShortcutsProps<TValue> extends Omit<ListProps, 'onChange'> {
+export interface ExportedPickersShortcutProps<TValue> extends Omit<ListProps, 'onChange'> {
   /**
    * Ordered array of shortcuts to display.
    * If empty, does not display the shortcuts.
    * @default `[]`
    */
   items?: PickersShortcutsItem<TValue>[];
+  eventImportance?: PickerChangeEventImportance;
```

Comment:
This would clearly need a JSDoc. 😄 
P.S. If we'd go with one of the suggested names for the variable, then probably removing `Event` from this type would also make sense 🤔

Reply:
Sure, I'm waiting for us to validate a naming and behavior and I will add a proper description here :angel: